File: app/src/main/java/org/aossie/agoraandroid/home/ReportBugFragment.java
package org.aossie.agoraandroid.home;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import org.aossie.agoraandroid.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class ReportBugFragment extends Fragment {

  public ReportBugFragment() {
  }

  @Override
  public View onCreateView(LayoutInflater inflater, ViewGroup container,
      Bundle savedInstanceState) {
    View view = inflater.inflate(R.layout.fragment_report_bug, null);
    Button openConsole = view.findViewById(R.id.button_report_bug);
    openConsole.setOnClickListener(new View.OnClickListener() {
      @Override
      public void onClick(View v) {
        startActivity(new Intent(Intent.ACTION_VIEW,
            Uri.parse("https://gitlab.com/aossie/agora-android/issues/new")));
      }
    });
    return view;
  }

  @Override
  public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
    super.onViewCreated(view, savedInstanceState);
  }
}

File: app/src/main/java/org/aossie/agoraandroid/home/ReportBugFragment.kt
package org.aossie.agoraandroid.home

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import androidx.fragment.app.Fragment
import org.aossie.agoraandroid.R

/**
 * A simple [Fragment] subclass.
 */
class ReportBugFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_report_bug, null)
        val openConsole = view.findViewById<Button>(R.id.button_report_bug)
        openConsole.setOnClickListener { startActivity(Intent(Intent.ACTION_VIEW, Uri.parse("https://gitlab.com/aossie/agora-android/issues/new"))) }
        return view
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }
}